src/Interfaces.ts

```diff
@@ -0,0 +1,5 @@
+
+export interface IExtension {
```

Comment:
```suggestion
export interface IExtension {
```

Or, in generally, run prettier :)